hourly_watch.py

```diff
@@ -15,6 +15,50 @@
 
 
 # Ska-specific watchers
+class SkaURLWatch(JobWatch):
+    def __init__(self, task, maxage_hours, url=None,):
+        self.type = 'URL'
+        self.basename = url
+        super(SkaURLWatch, self).__init__(task, url, maxage=maxage_hours * HOURS)
+
+    @property
+    def headers(self):
+        if not hasattr(self, '_headers'):
+            try:
+                import urllib2
+                response = urllib2.urlopen(self.basename)
+                self._headers = response.headers
+            except:
+                self._exists = False
+                self._headers = None
+        return self._headers
+
+    @property
+    def exists(self):
+        if not hasattr(self, '_exists'):
+            self._exists = self.headers is not None
+        return self._exists
+
+    @property
+    def age(self):
+        if not hasattr(self, '_age'):
+            if self.headers is not None:
+                import time
+                from Chandra.Time import DateTime
+                if 'Last-Modified' in self.headers:
+                    self.filetime = time.mktime(self.headers.getdate('Last-Modified'))
+                else:
+                    self.filetime = time.mktime(self.headers.getdate('Date'))
```

Comment:
Isn't `date` typically just the time the request was served?  Seems like that should have an `age` of `None` since you basically don't know anything about the actual file date.